lib/tasks/projects.rake

```diff
@@ -28,24 +28,28 @@ namespace :projects do
   task :check_status, %i[max_num_of_projects_to_check batch_size] => :environment do |_task, args|
     exit if ENV["READ_ONLY"].present?
 
-    # default should spread out around 50k package jobs per platform over ~ 83 minutes
-    max_num_of_projects_to_check = args.max_num_of_projects_to_check.nil? ? 50000 : args.max_num_of_projects_to_check.to_i
-    batch_size = args.batch_size.nil? ? 5000 : args.batch_size.to_i
-
-    PLATFORMS_FOR_STATUS_CHECKS.each do |platform|
```

Comment:
With the new prioritization method of looking at dates, it makes more sense to just look at all platform that are allowed simultaneously, rather than iterating over them.